lib/mjsonwp/routes.js

```diff
@@ -135,7 +135,21 @@ const METHOD_MAP = {
     GET: {command: 'getText'}
   },
   '/wd/hub/session/:sessionId/element/:elementId/value': {
-    POST: {command: 'setValue', payloadParams: {required: ['value']}}
+    POST: {command: 'setValue', payloadParams: {
+      validate: (jsonObj) => {
```

Comment:
I do when it can stay within 80 chars length or so. it was the line length that made me want to chop it